release-notes.md

```diff
@@ -0,0 +1,4 @@
+## Patient (zib-Patient)
```

Comment:
I believe the release notes miss a line for ZIB-2218 for MaritalStatus.

Perhaps add it like 
## MaritalStatus (zib-Patient, nl-core-Patient)
* xxx ; xxxx ; xxx

Reply:
En miss voor Nationality dan:

## Nationality (zib-Patient, nl-core-Patient)
* No significant changes.

Reply:
Missing release note for LanguageProficiency -  ZIB-1877: